www_src/components/select/select.less

```diff
@@ -0,0 +1,12 @@
+.select {
+  -webkit-appearance: none;
```

Comment:
hm, I was more wondering about whether we could add it in ourselves in the mean time. Can't seem to find useful info on that going through the various docs there, though.